src/librustc/ty/query/on_disk_cache.rs

```diff
@@ -943,6 +943,7 @@ where
 
 macro_rules! encoder_methods {
     ($($name:ident($ty:ty);)*) => {
+        #[inline]
         $(fn $name(&mut self, value: $ty) -> Result<(), Self::Error> {
```

Comment:
The `#[inline]` here is only applied to the first method. Always putting `$(` on its own line would have avoided this. cc rust-lang/rustfmt#8